### DISCUSSION TITLE
是否有打算添加发送专栏的api

### DISCUSSION THREAD

**Discussion by adk23333** [Nemo2011/bilibili-api#741] [Ideas]
[合并图文的议题](https://github.com/Nemo2011/bilibili-api/issues/662)

是暂时没时间添加还是这里面有坑，无坑的话，我去试试？

**Comment by z0z0r4** [comment]
发送专栏看起来会很麻烦，暂时没人做吧我记得

这个 issue 只是读取已有专栏的坑，无关发送

**Reply by z0z0r4** [reply]
欢迎但肉眼可见是个大活

**Reply by adk23333** [reply]
> 发送专栏看起来会很麻烦，暂时没人做吧我记得

我这边有需求并不想用drissionpage之类的实现，如果不是技术上有大坑的话，我先提个issue？后续我拉个pr

> 这个 issue 只是读取已有专栏的坑，无关发送

关联一下

**Reply by z0z0r4** [reply]
关联？欢迎pr

**Reply by adk23333** [reply]
> 关联？欢迎pr

方便快速找到所有相关话题的小技巧而已（手动狗头

修完我的tieba-admin项目的bug就pr一个

**Reply by z0z0r4** [reply]
贴吧更是一坨=-=

**Reply by adk23333** [reply]
> 贴吧更是一坨=-=

有aiotieba的作者在处理接口，我只需要调用，哈哈
